src/vscode-dts/vscode.d.ts

```diff
@@ -6178,6 +6178,8 @@ declare module 'vscode' {
 	 * Denotes a location of an editor in the window. Editors can be arranged in a grid
 	 * and each column represents one editor location in that grid by counting the editors
 	 * in order of their appearance.
+	 *
+	 * Columns that do not exists will be created as needed up to the maximum of `ViewColumn.Nine`.
```

Comment:
@bpasero I believe that comment makes sense but it shouldn't be here but in places where we create new columns, e.g for [`showTextDocument`](https://github.com/microsoft/vscode/blob/dced70bbf36d3c53c08e791da1791ac7fc42519b/src/vscode-dts/vscode.d.ts#L9454) which currently states the opposite of this

Reply:
👍 will tackle in a new PR.